OpenQA/Isotovideo/Utils.pm

```diff
@@ -119,4 +120,50 @@ sub checkout_git_refspec {
     calculate_git_hash($dir);
 }
 
+=head2 load_test_schedule
+
+Loads the test schedule (main.pm) or particular test modules if the `SCHEDULE` variable is
+present.
+
+=cut
+
+sub load_test_schedule {
+    # add lib of the test distributions - but only for main.pm not to pollute
+    # further dependencies (the tests get it through autotest)
+    my @oldINC = @INC;
+    unshift @INC, $bmwqemu::vars{CASEDIR} . '/lib';
+    if ($bmwqemu::vars{SCHEDULE}) {
+        diag 'Enforced test schedule by \'SCHEDULE\' variable in action';
+        $bmwqemu::vars{INCLUDE_MODULES} = undef;
+        autotest::loadtest($_ . '.pm') foreach split(',', $bmwqemu::vars{SCHEDULE});
+        $bmwqemu::vars{INCLUDE_MODULES} = 'none';
```

Comment:
just a question, I am wondering why `vars{INCLUDE_MODULES}` is set twice?

Reply:
don't worry. This code is just moved from https://github.com/os-autoinst/os-autoinst/pull/1409/files#diff-07616258592e792ab88f484012b38370L283 . The variable is evaluated within `autotest::loadtest` in there within `basetest::is_applicable`